### ISSUE TITLE
Examples are incorrect

### ISSUE THREAD

**Issue by Waxolunist** [oleics/node-filecache#2]
The examples as presented are not working. The options-object has to be provided on construction of filecache.

**Comment by cxcorp** [comment]
Can confirm, issue is still present.

```
<removed>\node_modules\filecache\filecache.js:179
      defaultOptions[k] = options[k]
                        ^

TypeError: Cannot set property 'watchFileChanges' of undefined
    at <removed>\node_modules\filecache\filecache.js:179:25
    at Array.forEach (native)
    at EventEmitter.em.options (<removed>\node_modules\filecache\fil
ecache.js:178:26)
    at Object.<anonymous> (<removed>\index.js:4:4)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.runMain (module.js:590:10)
```

**Comment by oleics** [comment]
Fixed in 437f619 and released in v0.2.2 (fa0fb42)
